File: 00_ETC/12756.cpp
#include <iostream>
using namespace std;

int main() {
    cin.tie(0)->sync_with_stdio(0);

    int playerAAtkPnt, playerBAtkPnt, playerALife, playerBLife;
    cin >> playerAAtkPnt >> playerALife;
    cin >> playerBAtkPnt >> playerBLife;

    while (true) {
        playerALife -= playerBAtkPnt;
        playerBLife -= playerAAtkPnt;
        if (playerALife <= 0 || playerBLife <= 0) {
            break;
        }
    }
    if (playerALife > 0 && playerBLife <= 0) {
        cout << "PLAYER A" << '\n';
    } else if (playerBLife > 0 && playerALife <= 0) {
        cout << "PLAYER B" << '\n';
    } else {
        cout << "DRAW" << '\n';
    }
    return 0;
}
//
// Created by 최승우 on 25. 9. 7.
//

File: 00_ETC_PY/12756.py
playerAAtkPnt, playerALife = map(int, input().split())
playerBAtkPnt, playerBLife = map(int, input().split())

while True:
    playerALife -= playerBAtkPnt
    playerBLife -= playerAAtkPnt
    if playerALife <= 0 or playerBLife <= 0:
        break
if playerALife > 0 and playerBLife <= 0:
    print("PLAYER A")
elif playerALife <= 0 and playerBLife > 0:
    print("PLAYER B")
else:
    print("DRAW")